kernel/kernel_drivers.go

```diff
@@ -92,20 +91,26 @@ func createFirmwareSymlinks(fwMount, fwDest string) error {
 		return err
 	}
 
+	fwTarget := filepath.Join(fwMount.TargetMntPt, "firmware")
```

Comment:
idea: maybe MountPoints should have CurrentPath and TargetPath methods that do filepath.Join?

Reply:
Done